playwright/fixtures/pages/fronts.ts

```diff
@@ -60,4 +60,15 @@ const frontWithPageSkin: Front = {
 	section: 'uk',
 };
 
-export { frontWithPageSkin, fronts, tagPages };
+// This is a tag front, not supported by DCR yet, we can use prod and
```

Comment:
Minor - this comment reads like it's incomplete or 'and' should be something else